SPECS/gettext/gettext.spec

```diff
@@ -15,6 +15,24 @@ These allow programs to be compiled with NLS
 (Native Language Support), enabling them to output
 messages in the user's native language.
 
+%package common-devel
+Summary: Common development files for %{name}
+# autopoint archive
+License: GPLv3+
+BuildArch: noarch
+
+%description common-devel
+This package contains common architecture independent gettext development files.
+
+%package libs
+Summary: Libraries for %{name}
+# libasprintf is LGPLv2+
```

Comment:
Please use the `License` tag instead - they can be separate for each subpackage.